specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/stable/2024-05-01/appconfiguration.json

```diff
@@ -0,0 +1,2890 @@
+          },
+          "description": "The list of deleted configuration store."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of deleted configuration stores."
+        }
+      },
+      "description": "List of deleted configuration stores"
+    },
+    "ReplicaListResult": {
+      "description": "The result of a request to list replicas.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The collection value.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Replica"
+          }
+        },
+        "nextLink": {
+          "description": "The URI that can be used to request the next set of paged results.",
+          "type": "string"
+        }
+      }
+    },
+    "Replica": {
+      "description": "The replica resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the replica.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
+          "description": "The location of the replica.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "Resource system metadata.",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/ReplicaProperties",
+          "description": "All replica properties.",
+          "x-ms-client-flatten": true,
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ReplicaProperties": {
+      "description": "All replica properties.",
+      "type": "object",
+      "properties": {
+        "endpoint": {
+          "description": "The URI of the replica where the replica API will be available.",
+          "type": "string",
+          "readOnly": true
+        },
+        "provisioningState": {
+          "description": "The provisioning state of the replica.",
+          "enum": [
+            "Creating",
+            "Succeeded",
+            "Deleting",
+            "Failed",
+            "Canceled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ReplicaProvisioningState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Snapshot": {
+      "description": "The snapshot resource.",
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
>id [](http://example.com/codeflow?start=9&length=2)

can you use arm-id as format for this property?

Reply:
Updated